### ISSUE TITLE
Push Notifications

### ISSUE THREAD

**Issue by jaullo** [cloverstudio/Spika-Server#81]
Hello people, 

Can some one give a hand to make push notification work? I installed the backend and all dependencies on windows server. App works ok (sing up, join, gropus, friends, chats) but push notifications are not. I setup a new project in google console and try to make it work on spika but there is no way. Google maps works fine with my new project data and keys.

Maybe we can work together to make it work, becaus we don`t have company support at all

Best regards

**Comment by jaullo** [comment]
I forgot to mention that i used the 1 to 6 guide found in this page but is not working.

**Comment by mediakliq** [comment]
They are now charging to fix this issue which tells me it was the plan from
the beginning.

**Comment by jaullo** [comment]
Oh, i see. I will try to fix it or rewrite push notifications code to make it work :)

**Comment by mediakliq** [comment]
Yes please do and report back and share your findings or fixes please thanks

**Comment by jaullo** [comment]
To keep things separated, I will start a new project based on spika to start over and fix some bugs that i found. I some one wants to join and help, please contact me at jason749_8@hotmail.com
Best regards

**Comment by mediakliq** [comment]
Ok that sounds great it's much needed! Keep us updated with your progress. Thanks

**Comment by jaullo** [comment]
Here is a work around to get push notifications
https://github.com/jaullo/Snappy/issues/1
